### DISCUSSION TITLE
Huawei sun2000 6KTL M1 with Luna2000 10kwh

### DISCUSSION THREAD

**Discussion by reneelout** [wlcrs/huawei_solar#710] [Setting up the integration]
Hello, I have the above configuration together with a Wallbox Pulsar Plus autocharger.
Because of the fact that more and more energy company's are charging for delivering energy back to the grid I would like to have an option (together with an automation) in Home Assistant/the integration that disables this. But I still would like maximum use of the solar panels and the battery. So disabling the inverter and/or the solar panels is not an option I think.

Is this an option within the integration?
On the web there are several documents, also from Huawei, but there options discussed that I don't have available.

I hope somebody can help me out here.

**Comment by Roving-Ronin** [comment]
@reneelout 
Whilst this integration can allow setting that function, calling it as a service, it is not required to do it. What your asking about is something generic to the Huawei inverters, suggest reading the manuals.

See Section 7.2.1 - https://myenergygroup.com.au/wp-content/uploads/2022/03/HUW-SUN2000-5KTL-M1-INSTALLATION-MANUAL.pdf

and/or https://photomate.zendesk.com/hc/en-gb/articles/5701625507485-Export-limitation-for-SUN2000-inverters-via-FusionSolar-App (refers to single phase, but merge with info from M1 3phase manual to complete).
